third_party/move/move-compiler-v2/src/lib.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright © Aptos Foundation
+// Parts of the project are originally copyright © Meta Platforms, Inc.
```

Comment:
From my reading of https://spdx.org/licenses/Apache-2.0.html, I think you need to retain the original notice on this file verbatim:

// Copyright (c) The Diem Core Contributors
// Copyright (c) The Move Contributors

and not change it to "Meta Platforms".  Adding a new Copyright in addition, as you have done, is fine, but seems more appropriate to be Aptos Labs than Aptos Foundation.